app/models/group.rb

```diff
@@ -590,9 +590,9 @@ def self.refresh_automatic_group!(name)
       when :staff
         "SELECT id FROM users WHERE id > 0 AND (moderator OR admin) AND NOT staged"
       when :trust_level_1, :trust_level_2, :trust_level_3, :trust_level_4
-        "SELECT id FROM users WHERE id > 0 AND trust_level >= #{id - 10} AND NOT staged"
+        "SELECT id FROM users WHERE id >= -1 AND trust_level >= #{id - 10} AND NOT staged"
       when :trust_level_0
-        "SELECT id FROM users WHERE id > 0 AND NOT staged"
+        "SELECT id FROM users WHERE id >= -1 AND NOT staged"
```

Comment:
Include the system user when adding users into TL groups.

Reply:
Yeah definitely need to include discobot (which is defined as `DiscourseNarrativeBot::BOT_USER_ID`)

It would be nicer if we had a way for plugins to register bot users in core...I think there may be an AI bot user too. I will go check